src/js/view/controls/controls.js

```diff
@@ -610,6 +616,11 @@ export default class Controls extends Events {
         this.showing = false;
         addClass(this.playerContainer, 'jw-flag-user-inactive');
         this.trigger('userInactive');
+
+        if (shouldFocusOnInactive) {
```

Comment:
How about implementing this as a `'userInactive'` listener instead of a flag to avoid side-effects? Ex:
```js
controlbar.on(USER_ACTION, () => {
    this.once('userInactive', () => this.focusPlayerElement(), this);
    this.userActive();
});
```
```js
const blurCallback = (evt) => {
    this.off('userInactive', null, this);
    const focusedElement = evt...
```

Reply:
this is a great idea